### PR TITLE
Make primary wallet implicit in UI

### PR DESCRIPTION
**Pull request by antonyshchenko**
https://github.com/user-attachments/assets/adca85e9-f8f0-450b-96cc-e1ac422f5d9c

### CODE REVIEW

File: web-ui/src/components/Screens/Header.tsx [hunk 1]
```diff
@@ -94,52 +69,104 @@ export function Header({
...
-                  )}
-                  {wallet.primaryCategory === 'evm'
-                    ? evmAddressDisplay(wallet.primaryAddress ?? '0x')
-                    : bitcoinAddressDisplay(wallet.primaryAddress)}
-                </span>
+            <Popover className="relative">
+              {() => (
+                <>
+                  <Popover.Button className="flex items-center gap-3 overflow-hidden text-ellipsis rounded-[20px] bg-darkBluishGray7 px-4 py-2 text-darkBluishGray1 transition-colors duration-300 ease-in-out hover:bg-primary4 focus:outline-none focus:ring-1 focus:ring-inset focus:ring-mutedGray">
+                    Connected wallets:
```
Comment (bholzman) [inline web-ui/src/components/Screens/Header.tsx:78]: I kind of think we can probably get away without saying anything here -- they just connected their wallet and there's a new header icon, I think people will figure out what it is for

**Review by bholzman — APPROVED** [review]
Looks great, nice cleanup!

File: web-ui/src/components/Screens/Header.tsx (hunk 1 above)
Comment (antonyshchenko) [reply web-ui/src/components/Screens/Header.tsx:78]: Idk, a button with icons and no text could looks weird, but I will try tomorrow

Comment (antonyshchenko) [reply web-ui/src/components/Screens/Header.tsx:78]: Tried without text and I'm not convinced. Going to leave it as is. Here are some screenshots:

Screenshot 2024-09-12 at 12 35 11
Screenshot 2024-09-12 at 12 35 41
Screenshot 2024-09-12 at 12 35 54
Screenshot 2024-09-12 at 12 36 20

**Pull request merged**